content/tutorials/single-page-tutorials/setup_iota-js_ for_development.md

```diff
@@ -0,0 +1,190 @@
+# New Tutorial - How to setup iota.js for development
+
+:::info
+
+In this tutorial you will learn how to create a new project, generate a new address and receive testnet funds from the faucet.
+
+:::
+
+
+## Prerequisites
+
+- Development Environment and Libraries
+  - Installed [Node.js](https://nodejs.org/en/)
+
+***
+
+## Setup
```

Comment:
```suggestion
## Set Up
```
You set up your setup (I know, it's confusing). But basically, if it's instructions it's Set Up, if it's describing the required "tools" it's setup